src/measure_theory/measure_space.lean

```diff
@@ -2239,6 +2242,10 @@ by simp [measure.trim]
 
 variables {m m0 : measurable_space α} {μ : measure α} {s : set α}
 
+lemma to_outer_measure_trim_eq_trim_to_outer_measure (μ : measure α) (hm : m ≤ m0) :
+  @measure.to_outer_measure _ m (μ.trim hm) = @outer_measure.trim _ m μ.to_outer_measure :=
```

Comment:
Are all the `@`s needed here?

Reply:
yes. Otherwise `m0` is incorrectly inferred. I could make `m` be the default one, but then the definition of `μ` would need an `@` to use `m0` and `μ.to_outer_measure` would need  `@` as well. I have to have them somewhere.